component/pyroscope/java/java.go

```diff
@@ -21,8 +22,9 @@ const (
 
 func init() {
 	component.Register(component.Registration{
-		Name: "pyroscope.java",
-		Args: Arguments{},
+		Name:      "pyroscope.java",
+		Stability: featuregate.StabilityStable,
```

Comment:
@thampiotr The docs say this is beta, not stable: https://grafana.com/docs/agent/latest/flow/reference/components/pyroscope.java/